### DISCUSSION TITLE
Is the Microsoft Store version better than the one in github releases?

### DISCUSSION THREAD

**Discussion by ShashTheEpik** [rocksdanister/lively#1345] [General]
Also, does the Microsoft Store one receive less frequent updates or is it the same?

**Comment by rocksdanister** [comment]
Both versions are usually same with some minor differences.
New versions will release in github first; store updates takes time (validation from Microsoft side.)

Further read:
https://github.com/rocksdanister/lively/wiki/Differences-Between-Distributions
